lti_consumer/lti_1p3/consumer.py

```diff
@@ -540,18 +555,57 @@ def enable_deep_linking(
         """
         self.dl = LtiDeepLinking(deep_linking_launch_url, deep_linking_return_url)
 
+    def enable_proctoring(
+        self,
+        attempt_number,
```

Comment:
TODO: Make `session_data` and `resource_link` positional arguments, as they're required.